jumpscale/sals/vdc/vdc.py

```diff
@@ -479,17 +474,24 @@ def show_vdc_payment(self, bot, expiry=5, wallet_name=None):
         else:
             return True, amount, payment_id
 
-    def show_external_node_payment(self, bot, size, no_nodes=1, expiry=5, wallet_name=None, public_ip=False):
+    def show_external_node_payment(self, bot, farm_name, size, no_nodes=1, expiry=5, wallet_name=None, public_ip=False):
```

Comment:
in general, it's bad practice to change the method signature.
the right way is to make it a keyword argument. I'm sure you already changed usage but for future reference